BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeHttpClient.kt

```diff
@@ -136,48 +136,6 @@ internal class BraintreeHttpClient(
         httpClient.sendRequest(request, callback)
     }
 
-    /**
-     * Makes a synchronous HTTP POST request to Braintree.
-     *
-     * @param path the path or url to request from the server via HTTP POST
-     * @param data the body of the post request
-     * @param configuration configuration for the Braintree Android SDK.
-     * @param authorization
-     * @return the HTTP response body
-     */
-    @Throws(Exception::class)
-    fun post(
```

Comment:
I'm assuming this is removed because we aren't using it? Just funny to see a post request removed but makes sense if we aren't using it anywhere.